### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by williamhickok11**
Here is my first draft of the quiz.

### CODE REVIEW

File: JS/augEventListeners.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+var CarLot = (function (inventory) {
+
+  inventory.activateEvents = function () {
+    var clickedEl;
+
+    //--- Access and edit description on the DOM --//
+    document.getElementsByClassName("description")[0].addEventListener("click", CarLot.editDescription);
```
Comment (chortlehoort) [inline JS/augEventListeners.js]: No hard-coded event listeners.  What happens when you add another car to your JSON file?  You'll have edit your source code every time.

File: JS/augEventListeners.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    document.getElementsByClassName("description")[1].addEventListener("click", CarLot.editDescription);
+    document.getElementsByClassName("description")[2].addEventListener("click", CarLot.editDescription);
+
+    //--- Event Listener on the Whole Container --//
+    document.getElementById("container").addEventListener("click", function(event){
+      console.log("event.target.className", event.target.className);
+
+      //--- Check where the mouse has clicked --//
+      if (event.target.className.indexOf("car") >= 0){
```
Comment (chortlehoort) [inline JS/augEventListeners.js]: Just use `event.target.getElementsByClassName()` here to eliminate the need for the `if..then` statements.

**Comment by chortlehoort** [comment]
@williamhickok11 Good start. As we discussed in person, make it a 3 column layout, and then check the comments I left.  If you have any questions about them, drop by and see me.

**Comment by gregkorte** [comment]
Nice job, William. Couple of things I see that are funky (I can break it). First I can only change the description when I click on the description. Not sure if this was something you wanted to bake in or not. Second, the previous change in the input does not go away when I click another box unless I click directly on the description. Last, I could not get the click button to do anything. Let me know if you have any questions.

**Comment by gregkorte** [comment]
Nice work, William. :+1: For bonus clear the input when you click the button and deselect the card.

**Pull request closed without merging**